lib/SampleService/core/config.py

```diff
@@ -163,11 +163,16 @@ def _check_string_req(s: Optional[str], name: str) -> str:
                             'type': 'object',
                             'properties': {
                                 'module': {'type': 'string'},
-                                'callable-builder': {'type': 'string'},
+                                'callable-builder': {'type': 'string'},  # TODO SCHEMA remove
+                                'callable_builder': {'type': 'string'},
                                 'parameters': {'type': 'object'}
                             },
                             'additionalProperties': False,
-                            'required': ['module', 'callable-builder']
+                            'required': ['module'],
+                            'oneOf': [
+                                {'required': ['callable_builder']},
+                                {'required': ['callable-builder']},
```

Comment:
is the plan to remove the support for "callable-builder" when the config is updated?

Reply:
Right.